pkg/jwtAuth/authConfig.go

```diff
@@ -0,0 +1,53 @@
+package jwtAuth
```

Comment:
```suggestion
package jwt_auth
```
same for folder name